ansible/README.md

```diff
@@ -17,7 +17,37 @@ Nothing to be done, Ansible is already installed during vagrant provisioning.
 You can skip setup and prereq steps as those have been done by vagrant for you.  
 You may jump directly to [Deploying Using CouchDB](#deploying-using-couchdb)
 
-#### Mac users
+#### Docker for Mac users
+Docker for Mac does not provide any official ways to fulfill some requirements for OpenWhisk.
+You need to depends on the workarounds until Docker provides official methods.
```

Comment:
Did you also consider the `docker-compose` method - https://github.com/openwhisk/openwhisk-devtools/tree/master/docker-compose ?